optics/optics.cabal

```diff
@@ -146,6 +146,9 @@ benchmark folds
   hs-source-dirs:   benchmarks
   ghc-options:      -Wall -threaded
 
+  if impl(ghc >= 8.8)
+    buildable: False
+
```

Comment:
What's going on here? Surely `optics` should be buildable on 8.8?

Reply:
lens doesn't build on 8.8.1

Reply:
Sorry, my bad, I missed that these were just specific components that depend on `lens`.

Reply:
`lens` will be buildable any moment now